src/__tests__/starWarsData.js

```diff
@@ -93,14 +93,9 @@ const droidData = {
  * These are Flow types which correspond to the schema.
  * They represent the shape of the data visited during field resolution.
  */
-export type Character = {
-  id: string,
-  name: string,
-  friends: Array<string>,
-  appearsIn: Array<number>,
-};
+export type CharacterData = HumanData | DroidData;
```

Comment:
I prefer without the `Data` suffix for readability

Reply:
No please, because of:
https://github.com/graphql/graphql-js/blob/eef0878d851f7b8a4392b031550a1080497c018c/src/__tests__/starWarsSchema.js#L12